src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -45,9 +47,7 @@ enum WarriorSpells
     SPELL_WARRIOR_COLOSSUS_SMASH_EFFECT             = 208086,
     SPELL_WARRIOR_EXECUTE                           = 20647,
     SPELL_WARRIOR_GLYPH_OF_THE_BLAZING_TRAIL        = 123779,
-    SPELL_WARRIOR_GLYPH_OF_HEROIC_LEAP              = 159708,
-    SPELL_WARRIOR_GLYPH_OF_HEROIC_LEAP_BUFF         = 133278,
-    SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 178368,
+    SPELL_WARRIOR_HEROIC_LEAP_JUMP                  = 94954, // jump + dmg
```

Comment:
i don't think this spellid is right

Reply:
yes its not 
correct spell ids:
id_1 = 162052 Jumps
id_2 = 52174 Doing Damage on landing
but this one do both work you can test and see

Reply:
@defsky : this PR got closed because @Hadifaren deleted their PR branch [on Nov 21, 2017](https://github.com/TrinityCore/TrinityCore/pull/20839#event-1351706963) and a new PR must be opened for any of this to be available for further work.